components/Post/Post.jsx

```diff
@@ -289,8 +289,7 @@ export default class Post extends React.Component {
                 </View>
                 <View style={styles.headerBody}>
                   <View style={styles.authorDetails}>
-                    <Text>{authorName}</Text>
-                    <Text>{this.props.showTag ? ` ►  ${tags[0]}` : null}</Text>
+                    <Text>{authorName}{this.props.showTag ? ` ►  ${tags[0]}` : null}</Text>
```

Comment:
Hello darkness my old friend... Reason I made these separate components was so that the channel name could overflow onto the line below if the name + tag was too long (it would run off the screen). Found it harder than it should have been to make it behave and wonder if this change will annul its ability to line-break properly. With that said, there should be a way to simply allow overflow within the `<Text>` element and constrain it to the screen edge, I just couldn't find the magic words when working on it. Lmk if you've tested that and it's worked though :)